bot/resources/evergreen/py_topics.yaml

```
@@ -16,23 +14,34 @@
     - What feature do you think should be added to Python?
     - Has Python helped you in school? If so, how?
     - What was the first thing you created with Python?
+    - What is your favorite Python package?
+    - What standard library module is really underrated?
+    - Have you published any packages on PyPi? If so, what are they?
+    - What are you currently working on in Python?
+    - What's your favorite script and how has it helped you in day to day activities?
+    - When you were first learning, what is something that stumped you?
+    - When you were first learning, what is a resource you wish you had?
+    - What is something you know now that you wish you knew when you were starting out?
```

The seems quite wordy to me, maybe a comma could help?
```suggestion
    - What is something you know now, that you wish you knew when you were starting out?
```

Maybe even
```suggestion
    - What is something you know now, that you wish you knew when starting out?
```